lib/plugins/openaps.js

```diff
@@ -6,6 +6,8 @@ var times = require('../times');
 var levels = require('../levels');
 var timeago = require('./timeago')();
 
+var ALL_STATUS_FIELDS = ['status-symbol', 'status-label', 'iob', 'meal-assist', 'freq', 'rssi'];
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["ALL_STATUS_FIELDS" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2314040627/issues/source?bid=2452379&fileBranchId=3200266#l9)